# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/websockets/cookies/__dir__.ini
# Commit: 39f2e21623aa
# Full Hash: 39f2e21623aa7c19517e32cbd97dc1c3584c58ca
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2020-06-02 03:24:46
# Regressor Bug: 1638358
# File Overlap Count: 1
# Description:
#   Bug 1638358 - Cookie Schemeful Same-Site - part 5 - schemeful comparison, r=mayhemer
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D75629
# ==============================================================================

diff -r 2c873c72bf1f -r 39f2e21623aa testing/web-platform/meta/websockets/cookies/__dir__.ini
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/testing/web-platform/meta/websockets/cookies/__dir__.ini	Mon Jun 01 16:49:03 2020 +0000
@@ -0,0 +1,1 @@
+prefs: [network.cookie.sameSite.schemeful:false]
